src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -98,6 +98,17 @@ def test_compile_model_without_device(request, tmp_path):
     assert isinstance(compiled_model, CompiledModel)
 
 
+# request - https://docs.pytest.org/en/7.1.x/reference/reference.html#request
+def test_compile_model_with_auto_plugin(request, tmp_path):
```

Comment:
Could you please add another test for `openvino.compile_model()` in which you give non-default `device_name` and `config` arguments? `device_name` can be just `CPU`.

Long story short, it would be perfect to have the new case covered for both default and non-default use cases.